src/components/LocationDetails.js

```diff
@@ -1,24 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import "../styles/LocationDetails.css";
-
-function LocationDetails(props) {
```

Comment:
You can destructure in the function definition like this:
`function LocationDetails({ city, country, errorMessage }) {`